programming/sr/ruggeduinos/custom_firmware.md

```diff
@@ -12,31 +12,15 @@ This documentation refers to a feature which is only available on the physical r
 
 The Ruggeduino that came as part of your kit was shipped with a firmware that provides the functionality outlined in the [Ruggeduino](/docs/programming/sr/ruggeduinos) page.
 You may wish to extend the functionality of this firmware, or completely replace it.
-The `sr.robot` library provides support for three Ruggeduino firmware scenarios:
+The `sr.robot3` library provides support for three Ruggeduino firmware scenarios:
 
  1. Default SR firmware
  2. [Extended SR firmware](#extension): Firmwares that add commands to the default SR firmware.
  3. [Completely custom](#completely): Any firmware not derived from the SR firmware.
 
-By default, the [`sr.robot`](/docs/programming/sr/) library assumes that all connected Ruggeduinos are running the SR firmware.
+By default, the [`sr.robot3`](/docs/programming/sr/) library assumes that all connected Ruggeduinos are running the SR firmware.
 If you wish to use an extended SR firmware, or completely custom firmware,
  then you need to tell the `Robot` object what to do with your Ruggeduino(s).
-To do this, you will need to expand the initialisation of your `Robot` object as detailed [here](/docs/programming/sr/#CustomRobotInit).
```

Comment:
Please can we include examples of how to interact with custom firmware? I realise the API is different, but it doesn't feel great to be saying that something can be done without explaining how.

Reply:
Ah, I see this is mentioned a bit later. Perhaps here just summarise it a bit? Otherwise you need to read the whole page to understand what's needed. Alternatively, perhaps reword this sentence to something like:
> If you're using completely custom firmware, you'll need to tell the kit to ignore the ruggeduino so that you're able to define your own setup logic.

Reply:
Even just adding a "this is covered in section ..." would solve this.

Reply:
210a285